.travis/osx..install.sh

```diff
@@ -2,18 +2,18 @@
 
 PACKAGES="cmake pkgconfig fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
 	PACKAGES="$PACKAGES homebrew/versions/qt55"
 else
 	PACKAGES="$PACKAGES cartr/qt4/qt"
 fi
 
-brew install $PACKAGES ccache
+brew install "$PACKAGES" ccache
 
 # Recompile fluid-synth without CoreAudio per issues #649
 # Changes to fluid-synth.rb must be pushed to URL prior to use
 url=$(git remote get-url origin)
 branch=$(git symbolic-ref --short HEAD)
-brew install --build-from-source $url/raw/$branch/cmake/apple/fluid-synth.rb
+brew install --build-from-source "$url"/raw/"$branch"/cmake/apple/fluid-synth.rb
```

Comment:
Actually, there's only two places where there is many var in the same path. Here and [there](https://github.com/LMMS/lmms/pull/3455/files#diff-e2415930ea7e22e5bcb8d5016798757dR48).